context_chat_backend/chain/one_shot.py

```diff
@@ -5,7 +5,7 @@
 from .context import ScopeType, get_context_chunks, get_context_docs
 from .query_proc import get_pruned_query
 
-_LLM_TEMPLATE = '''Answer based only on this context and do not add any imaginative details:
+_LLM_TEMPLATE = '''Answer based only on this context and do not add any imaginative details. Make sure to use the same language as the question in your answer.
 {context}
 
 {question}
```

Comment:
Adding `'''  # noqa: E501` near the end of the string should let the pre-commit workflow pass

Reply:
This chain is not used at all anyway though, right?

Reply:
This is the only chain we have. The linked prompt is a fallback if no prompt is given in the config.